frame/bags-list/src/benchmarks.rs

```diff
@@ -144,6 +144,10 @@ frame_benchmarking::benchmarks! {
 		// - both heavier's `prev` and `next` are nodes that will need to be read and written.
 		// - `lighter` is the bag's `head`, so the bag will need to be read and written.
 
+		// clear any pre-existing storage.
+		// NOTE: safe to call outside block production
```

Comment:
interesting, because in the genesis we put some stuff in there and this causes issues. 

@emostov perhaps this should be called in all benchmarks, specually ones where we use the `ListScenario`?

Reply:
It is being called in the other ones in the beginning as well, probably an oversight that it was not the case here.